src/sources/readerthread.cpp

```diff
@@ -39,7 +39,6 @@ ReaderThread::ReaderThread( QObject *receiver ) :
   m_id( 0 ),
   m_numValues( 1 )
 {
-  m_buffer[14] = '\0';
```

Comment:
Why do you remove this line?

Reply:
m_buffer is redundant when QByteArray dataOut is built from iterating over m_fifo. It caused errors when passing binary data though the slot.